_includes/settlement-reconciliation.md

```diff
@@ -211,6 +213,17 @@ the reconciliation file.
 *   `capture.transaction.number` returned from Swedbank Pay is equal to
     `TransactionNo` in reconciliation file.
 
+Below you will see the API mapping tables to the fields in the PBA reports for
+`Create payment`, `Capture` and `Reversal`.
```

Comment:
This description should be different based on the `documentation_section` as `swish` doesn't have `Capture`.

Reply:
fixed!